components/validator/validator.go

```diff
@@ -86,7 +87,9 @@ func NewValidator(parentCtx context.Context, cfg Config, l log.Logger, m metrics
 
 	ctx, cancel := context.WithCancel(parentCtx)
 
-	l2OutputSubmitter, err := NewL2OutputSubmitter(cfg, l)
+	txCandidatesChan := make(chan txmgr.TxCandidate, 10)
```

Comment:
Is there any possibility that it sends through the channel more than 10? How about injecting the value from the config or something?

Reply:
Output submission transaction is sent hourly (if selected next validator always), and challenge is a rare case, so I think it would be ok now. wdyt?

Reply:
I was just worried that there's case this number is not enough for e2e or something. Let's tackle the issue when we face it!